Maintenance/test_handling/create_testresult_page

```diff
@@ -450,9 +450,7 @@ sub print_platform_descriptions()
 <th>BOOST</th>
 <th>MPFR</th>
 <th>GMP</th>
-<th>QT3</th>
-<th>QT4</th>
-<th>OPENGL</th>
+<th>QT5</th>
```

Comment:
That line is about the display, in the test results, of the Qt3 version. Nobody cares of the precise Qt3 version that is installed. Either Qt3 is installed and configured in CMake, and in that case that is probably Qt-3.8.x, or it is not installed or configured. We can see the absence of Qt3 by the blue cells in the corresponding `*_Demo` lines.

I can restore the display of the Qt3 version, but I do not think that is necessary.